examples/gallery/images/grdlandmask.py

```diff
@@ -0,0 +1,25 @@
+"""
+Create 'wet-dry' mask grid
+--------------------------
+The :meth:`pygmt.grdlandmask` method allows setting 
+all nodes on land or water to a specified value using
+the ``maskvalues`` parameter. 
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+
+# Define region of interest
+region = [-65, -40, -40, -20]
+
+# Assign a value of 0 for all water masses and a value of 1 for all land masses
```

Comment:
Also need to explain the `spacing` and `resolution` parameters.

Reply:
Done!